src/plugins/intel_gpu/src/graph/impls/onednn/deconvolution_onednn.cpp

```diff
@@ -35,6 +35,14 @@ static std::shared_ptr<dnnl::deconvolution_forward::primitive_desc> get_deconvol
     auto output_md = onednn::layout_to_memory_desc(output_layout, tag_in_out);
     auto grouped_weights = format::is_grouped(weights_layout.format) || prim->grouped_weights_shape;
 
+    // Extend deconv parameters in case if spatials rank of output memory doesn't match size of parameters
```

Comment:
Most likely an error in the `dilation` initialization
`dnnl::memory::dims dilation(input_layout.get_spatial_rank(), 1);`
For some reason dilations from the primitive are not used
Maybe try to take it from primitive, or initialize it relative to the rank of the stride or paddings?